[2023-11-23T22:12:29.695+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-21T00:00:00+00:00 [queued]>
[2023-11-23T22:12:29.707+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-21T00:00:00+00:00 [queued]>
[2023-11-23T22:12:29.707+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-11-23T22:12:29.726+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract> on 2023-11-21 00:00:00+00:00
[2023-11-23T22:12:29.732+0000] {standard_task_runner.py:57} INFO - Started process 142 to run task
[2023-11-23T22:12:29.737+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'extract', 'scheduled__2023-11-21T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpbkq_sq67']
[2023-11-23T22:12:29.741+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask extract
[2023-11-23T22:12:29.795+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.extract scheduled__2023-11-21T00:00:00+00:00 [running]> on host 3c6bdd9606b4
[2023-11-23T22:12:30.081+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2023-11-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-21T00:00:00+00:00'
[2023-11-23T22:12:33.340+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-23T22:12:33.351+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=outcomes_dag, task_id=extract, execution_date=20231121T000000, start_date=20231123T221229, end_date=20231123T221233
[2023-11-23T22:12:33.385+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-23T22:12:33.426+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-24T21:55:13.260+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-21T00:00:00+00:00 [queued]>
[2023-11-24T21:55:13.282+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-21T00:00:00+00:00 [queued]>
[2023-11-24T21:55:13.283+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-11-24T21:55:13.461+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract> on 2023-11-21 00:00:00+00:00
[2023-11-24T21:55:13.471+0000] {standard_task_runner.py:57} INFO - Started process 209 to run task
[2023-11-24T21:55:13.476+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'extract', 'scheduled__2023-11-21T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpyqr7tyi0']
[2023-11-24T21:55:13.480+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask extract
[2023-11-24T21:55:13.543+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.extract scheduled__2023-11-21T00:00:00+00:00 [running]> on host 30eca782d34c
[2023-11-24T21:55:13.646+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2023-11-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-21T00:00:00+00:00'
[2023-11-24T21:55:14.566+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/etl_scripts/extract.py", line 35, in extract_data
    storage_client = storage.Client('oceanic-hangout-406022')
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/storage/client.py", line 172, in __init__
    super(Client, self).__init__(
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/client/__init__.py", line 321, in __init__
    Client.__init__(
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/client/__init__.py", line 178, in __init__
    credentials, _ = google.auth.default(scopes=scopes)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/auth/_default.py", line 657, in default
    credentials, project_id = checker()
  File "/home/airflow/.local/lib/python3.8/site-packages/google/auth/_default.py", line 650, in <lambda>
    lambda: _get_explicit_environ_credentials(quota_project_id=quota_project_id),
  File "/home/airflow/.local/lib/python3.8/site-packages/google/auth/_default.py", line 270, in _get_explicit_environ_credentials
    credentials, project_id = load_credentials_from_file(
  File "/home/airflow/.local/lib/python3.8/site-packages/google/auth/_default.py", line 114, in load_credentials_from_file
    raise exceptions.DefaultCredentialsError(
google.auth.exceptions.DefaultCredentialsError: File opt/***/dags/etl_scripts/oceanic-hangout-406022-fd958fa4d302.json was not found.
[2023-11-24T21:55:14.581+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=outcomes_dag, task_id=extract, execution_date=20231121T000000, start_date=20231124T215513, end_date=20231124T215514
[2023-11-24T21:55:14.597+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 13 for task extract (File opt/***/dags/etl_scripts/oceanic-hangout-406022-fd958fa4d302.json was not found.; 209)
[2023-11-24T21:55:14.616+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-24T21:55:14.637+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-24T23:01:57.999+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-21T00:00:00+00:00 [queued]>
[2023-11-24T23:01:58.026+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.extract scheduled__2023-11-21T00:00:00+00:00 [queued]>
[2023-11-24T23:01:58.027+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-11-24T23:01:58.070+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract> on 2023-11-21 00:00:00+00:00
[2023-11-24T23:01:58.079+0000] {standard_task_runner.py:57} INFO - Started process 108 to run task
[2023-11-24T23:01:58.091+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'extract', 'scheduled__2023-11-21T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpfj33epp2']
[2023-11-24T23:01:58.105+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask extract
[2023-11-24T23:01:58.344+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.extract scheduled__2023-11-21T00:00:00+00:00 [running]> on host 69a30b58663a
[2023-11-24T23:01:58.690+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2023-11-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-21T00:00:00+00:00'
[2023-11-24T23:01:59.938+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-24T23:02:00.165+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=outcomes_dag, task_id=extract, execution_date=20231121T000000, start_date=20231124T230158, end_date=20231124T230200
[2023-11-24T23:02:00.690+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-24T23:02:00.757+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
